scripts/quests/ahtUrhgan/Promotion_Private_First_Class.lua

```diff
@@ -1,15 +1,15 @@
 -----------------------------------
--- Assault PFC rank-up quest
-
+-- Promotion: Private First Class
+-- Log ID: 6, Quest ID: 90
 -- Naja Salaheem !pos 26 -8 -45.5 50
 -----------------------------------
-require('scripts/globals/items')
-require('scripts/globals/quests')
-require('scripts/globals/npc_util')
-require('scripts/globals/interaction/hidden_quest')
+require("scripts/globals/items")
```

Comment:
Just for the sake of consistency, could you move these to single quotes? Been trying to establish that pattern in quest and mission scripts